pkg/prowgen/prowgen.go

```diff
@@ -122,7 +122,12 @@ func GenerateJobs(configSpec *cioperatorapi.ReleaseBuildConfiguration, info *Pro
 	presubmits := map[string][]prowconfig.Presubmit{}
 	postsubmits := map[string][]prowconfig.Postsubmit{}
 	var periodics []prowconfig.Periodic
-
+	var jobRelease string
+	if configSpec.Releases != nil {
+		if release, found := configSpec.Releases["latest"]; found && release.Candidate != nil {
```

Comment:
You don't need the nilcheck, readaccess to a nil map is fine, it will just give `false` for the found bool.

Generally the PR looks fine but I do not know if inferring the release from here is correct, I would have thought `latest` is the `latest` release, not the one the job is targeting.

Reply:
As per discussion with @stevekuznetsov - "We need to read the value of releases.latest.version - if present - and use it to populate the job-release field when generating jobs"

Reply:
Yeah the version of the latest release we're importing is the definition of what this lablel should be